src/lib/viewers/media/DashControls.scss

```diff
@@ -29,6 +29,12 @@
     }
 }
 
+.bp-is-small-width-video {
+    .bp-DashControls-group--play-pause {
```

Comment:
Is it possible to conditionally render this in jsx instead of css hiding? I believe these controls are still in the DOM with css hiding. Conditional rendering may be better here for memory on mobile devices? May also be easier to test.